src/app/services/config/editor.config.json

```diff
@@ -221,12 +221,26 @@
     ]
   },
   "CONTENT_TYPES": {
-    "collection": {
+    "book": {
```

Comment:
This is not going to scale or work in future if we adopt for different framework data which is not specific to Education domain. 
Try to avoid Book, Course etc..